products/_bundle/templates/puppet/README.md.erb

```diff
@@ -49,6 +49,21 @@ Puppet module installer:
 
     puppet module install google/cloud
 
+Since these modules depend on the `googleauth` and `google-api-client` gems,
+you will also need to install those, with
+
+    /opt/puppetlabs/puppet/bin/gem install googleauth google-api-client
+
+If you prefer, you could also add the following to your puppet manifest:
+
+		package { [
+				'googleauth',
+				'google-api-client',
+			]:
```

Comment:
nit: this alignment looks weird to me. The ] is lining up with the 'a' in package, which seems like probably not the right place for it (same in other files too)

Reply:
I think it's correct - that's one more indent than `package`, which seems right to me.